src/Battlescape/ExplosionBState.cpp

```diff
@@ -110,16 +110,20 @@ void ExplosionBState::init()
 			_damageType = itemRule->getDamageType();
 		}
 
-		if (type == BT_PSIAMP || _hit)
+		if ((type == BT_PSIAMP
+			 || (type == BT_SCRIPTED
+				  && (action == BA_MINDCONTROL || action == BA_PANIC))) || _hit)
```

Comment:
why is BA_USE not allowed here?

Reply:
Because BA_USE was used as a special untargeted action in this proposal.